--- configure.orig	2024-12-03 12:00:00.000000000 +0000
+++ configure	2025-01-01 00:00:00.000000000 +0000
@@ -3265,7 +3265,7 @@

    if test "$cross_compiling" = yes; then :
   { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cross build not supported
-See \`config.log' for more details" "$LINENO" 5; }
+printf "%s\n" "$as_me: warning: allowing cross build for $host" >&2;}
+:; }
 fi
